recipe-server/client/control/components/action_fields/HeartbeatFields.js

```diff
@@ -117,6 +121,6 @@ export class HeartbeatFields extends ActionFields {
 
 export default connect(
   state => ({
-    recipeArguments: selector(state, 'arguments'),
+    recipeArguments: selector(state, 'arguments') || {},
```

Comment:
The changes in this file are unrelated to preference rollouts, right? I like the changes, but I don't think they belong in this PR. If nothing else, they are a source of conflict when we merge the feature branch in.